src/components/map/Map.js

```diff
@@ -152,20 +170,40 @@ class Map extends Component {
         };
 
         return (
-            <div ref={node => (this.node = node)} style={style}>
-                {layers.filter(layer => layer.isLoaded).map((config, index) => {
-                    const Overlay = layerType[config.layer] || Layer;
-
-                    return (
-                        <Overlay
-                            key={config.id}
-                            index={index}
-                            openContextMenu={openContextMenu}
-                            {...config}
-                        />
-                    );
-                })}
-                <Layer key="basemap" {...basemapConfig} />
+            <div
+                className={isDownload ? 'dhis2-maps-download ' : null}
```

Comment:
(from above) Why is `dhis2-maps-download` a global css style?

Reply:
Global css style replaced with styles object.